dockstore-webservice/src/main/java/io/dockstore/webservice/resources/UserResource.java

```diff
@@ -653,6 +657,49 @@ public Limits setUserLimits(@ApiParam(hidden = true) @Auth User authUser,
         return limits;
     }
 
+    @POST
+    @Path("/{userId}/services/refresh")
+    @Timed
+    @UnitOfWork
+    @ApiOperation(value = "Syncs service data with Git accounts.", notes = "Currently only works with GitHub", authorizations = {
+            @Authorization(value = JWT_SECURITY_DEFINITION_NAME) },
+            response = Workflow.class, responseContainer = "List")
+    public List<Workflow> syncUserServices(@ApiParam(hidden = true) @Auth User authUser,
+            @ApiParam(value = "User ID", required = true) @PathParam("userId") Long userId) {
+        final User user = userDAO.findById(authUser.getId());
```

Comment:
Are we going to add the ability for admins to refresh other people's services?

Reply:
It seems the intent for refresh workflows is to allow this, so I copied the intent here. It's not actually working correctly for workflows, where I created an issue for it, #2664, but I implemented it correctly it here, I think.

It does seem a little odd to me, so I'm open to removing it, in which case we should remove the userid from the path.

Reply:
I think we must have wrote that ability to potentially allow curators to refresh workflows if the "owners" of the workflow left their actual physical organization and left a workflow effectively owner-less. 
With webhooks, this concern should go away so I think this can be removed to simplify things

Reply:
Removed the ability for admin to update another user's services. Updated Path annotation so userId is no longer part of the url. Commit ad9d2614dcd987619a08015689e1fa117439eb62 (not yet pushed as of this writing)